### PR TITLE
Add doc for collecting ServiceHub logs

### PR DESCRIPTION
**Pull request by RyanToth3**
Add section in this doc for collecting additional information for ServiceHub related issues.

### CODE REVIEW

**Comment by PRMerger17** [comment]
@RyanToth3 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by madskristensen** [comment]
#sign-off

**Comment by PRMerger17** [comment]
@madskristensen: I'm sorry - only the author of this article, @seaniyer, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

File: docs/ide/how-to-increase-chances-of-performance-issue-being-fixed.md [hunk 1]
```diff
@@ -238,6 +240,21 @@ tool can be found on the [Recording performance traces with
...
+
+What makes these types of issues most actionable is to provide additional logs that can be collected by following these steps:
+
+1.  If this is a directly reproducible issue, start by deleting the %temp%/servicehub/logs folder. If you cannot reproduce this issue please keep this folder intact and ignore the following bullets:
+
+    -   Set the global environment variable ServiceHubTraceLevel to All
+    -   Reproduce the issue.
+
+2.  Download the Microsoft Visual Studio and .NET Framework Log Collection Tool [here](https://aka.ms/vscollect).
+3.  Run the tool. This outputs a zip file to %temp%/vslogs.zip. Please attach that file to your feedback.
```
Comment (sharwell) [inline docs/ide/how-to-increase-chances-of-performance-issue-being-fixed.md]: 💡 

```suggestion
3.  Run the tool. This outputs a zip file to **%temp%/vslogs.zip**. Please attach that file to your feedback.
```

File: docs/ide/how-to-increase-chances-of-performance-issue-being-fixed.md [hunk 2]
```diff
@@ -238,6 +240,21 @@ tool can be found on the [Recording performance traces with
 PerfView](https://github.com/dotnet/roslyn/wiki/Recording-performance-traces-with-PerfView)
 page.
 
+## OOP Issues
+
+There are a number of satellite processes that run parallel to Visual Studio and provide various features from outside of the main Visual Studio process. If an error occurs in one of these satellite processes it is usually seen on the Visual Studio side as a StreamJsonRpc.RemoteInvocationException or a StreamJsonRpc.ConnectionLostException.
```
Comment (sharwell) [inline docs/ide/how-to-increase-chances-of-performance-issue-being-fixed.md]: 💡 

```suggestion
There are a number of satellite processes that run parallel to Visual Studio and provide various features from outside of the main Visual Studio process. If an error occurs in one of these satellite processes it is usually seen on the Visual Studio side as a `StreamJsonRpc.RemoteInvocationException` or a `StreamJsonRpc.ConnectionLostException`.
```

File: docs/ide/how-to-increase-chances-of-performance-issue-being-fixed.md [hunk 3]
```diff
@@ -238,6 +240,21 @@ tool can be found on the [Recording performance traces with
...
+
+What makes these types of issues most actionable is to provide additional logs that can be collected by following these steps:
+
+1.  If this is a directly reproducible issue, start by deleting the %temp%/servicehub/logs folder. If you cannot reproduce this issue please keep this folder intact and ignore the following bullets:
```
Comment (sharwell) [inline docs/ide/how-to-increase-chances-of-performance-issue-being-fixed.md]:
```suggestion
1.  If this is a directly reproducible issue, start by deleting the **%temp%/servicehub/logs** folder. If you cannot reproduce this issue please keep this folder intact and ignore the following bullets:
```

File: docs/ide/how-to-increase-chances-of-performance-issue-being-fixed.md [hunk 4]
```diff
@@ -238,6 +240,21 @@ tool can be found on the [Recording performance traces with
...
+
+    -   Set the global environment variable ServiceHubTraceLevel to All
```
Comment (sharwell) [inline docs/ide/how-to-increase-chances-of-performance-issue-being-fixed.md]:
```suggestion
    -   Set the global environment variable **ServiceHubTraceLevel** to **All**.
```

File: docs/ide/how-to-increase-chances-of-performance-issue-being-fixed.md [hunk 5]
```diff
@@ -238,6 +240,21 @@ tool can be found on the [Recording performance traces with
 PerfView](https://github.com/dotnet/roslyn/wiki/Recording-performance-traces-with-PerfView)
 page.
 
+## OOP Issues
```
Comment (sharwell) [inline docs/ide/how-to-increase-chances-of-performance-issue-being-fixed.md]: 💡 Consider spelling OOP out as Out-Of-Process

Comment (sharwell) [inline docs/ide/how-to-increase-chances-of-performance-issue-being-fixed.md]: 💡 If a particular version of Visual Studio adds these logs automatically, consider adding a note like the following:

```
> :bulb: Starting with Visual Studio 2019 version ______, out-of-process logs are automatically attached to feedback submitted using the Report a Problem tool.
```

If these logs are only attached following a recording, you could add the following sentence as well:

> To submit feedback using this functionality, use the Report a Problem tool and make sure to Record the problem steps before submitting the feedback.

**Comment by PRMerger19** [comment]
@RyanToth3 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by mavasani** [comment]
Tag @gewarren @ghogen in case they can assist in the review.

**Comment by ghogen** [comment]
#sign-off

**Comment by PRMerger6** [comment]
@ghogen: I'm sorry - only the author of this article, @seaniyer, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Pull request merged**